README.md

```diff
@@ -1,25 +1,45 @@
-# fastcrypto
-Common cryptographic library used in software at Mysten Labs.
+# [fastcrypto]
+
+[![crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+![Apache2/MIT licensed][license-image]
+![Rust Version][rustc-image]
+
+fastcrypto` is a common cryptography library used by software at Mysten Labs. It is a wrapper library around several carefully selected crates with following considerations: 
```

Comment:
Please add "but on purpose published independently encouraging reusability across different applications and domains"